res/res_pjsip/pjsip_config.xml

```diff
@@ -681,6 +681,13 @@
 						dialplan application such as <emphasis>Ringing</emphasis>.</para>
 					</description>
 				</configOption>
+				<configOption name="tenantid" default="">
+					<synopsis>The tenant ID for this endpoint.</synopsis>
+					<description><para>
+						Sets the tenant ID for this endpoint. It will be present when
```

Comment:
Needed to clarify
 - what channel variable will have this value
 - what can be changed via dialplan... endpoint option tenantid or channel variable tenantid.
    For example:
    "Sets the tenant ID for this endpoint. The channel variable tenantid will have this value set on channel creation, and the channel variable tenantid can be changed later through dialplan."